defaults/main.yml

```diff
@@ -45,6 +45,9 @@ mysql_users: []
 
 install_rpm_repositories: "true"
 
+# To disable log_bin in percona >=8, enabled by default
+mysql_disable_log_bin: "true"
```

Comment:
Looks like you're defining the boolean as **mysql_**disable_log_bin here, but referring to it without the mysql_ preamble (i.e. matching the actual my.cnf clause) in the template files?